srcpkgs/dqlite/patches/musl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git src/vfs.c src/vfs.c
+index 31e138c..5d2a7c4 100644
+--- src/vfs.c
++++ src/vfs.c
+@@ -1,6 +1,7 @@
+ #include <string.h>
+ #include <sys/time.h>
+ #include <errno.h>
++#include <pthread.h>
```

Comment:
While cross-building for musl I didn't see this needed.

Reply:
I submitted it upstream. I think it was released and merged.